org.adempiere.base/src/org/compiere/tools/FileUtil.java

```diff
@@ -431,11 +436,77 @@ public static String getTempMailName(String subject, String extension) {
 		SimpleDateFormat sdf = new SimpleDateFormat("yyyyMMddHHmmss");
 		String dt = sdf.format(cal.getTime());
 		String cleanName = subject.replaceAll("[ &/]", "");
-		String localFile = System.getProperty("java.io.tmpdir")
-				+ System.getProperty("file.separator") + cleanName + "_" + dt + "_" + Env.getContext(Env.getCtx(), Env.AD_SESSION_ID)
-				+ extension;
+		String dir = System.getProperty("java.io.tmpdir")
+				+ System.getProperty("file.separator") + "rpttmp_" + dt + "_" + Env.getContext(Env.getCtx(), Env.AD_SESSION_ID) + System.getProperty("file.separator");

```

Comment:
another change (the use of hardcoded rpttmp_ prefix) that is not documented in the ticket.